### ISSUE TITLE
extra sqlalchemy engine args

### ISSUE THREAD

**Issue by DanCardin** [dgilland/sqlservice#5]
For example, I wanted to pass custom json en/decoder classes. At the moment, I just subclassed it to accept underscore prefixed args as ones which should not be mapped

```python
class SQLClient(sqlservice.SQLClient):
    def make_engine_options(self):
        result = super().make_engine_options()

        private_sqla_options = {o[1:]: v for o, v in self.config.items() if o.startswith('_')}
        result.update(private_sqla_options)
        return result
```
(though that was just to make it easier for me to visually see what the difference was, and to make as small a change as possible)

**Comment by dgilland** [comment]
Implemented in e0345a0789e726b1c020d0b9a42ec2392910888e.

You'll be able to do this instead:

```python
db = SQLClient(..., engine_options={...})
```

Will be included in the next `0.10.0` release.

**Comment by dgilland** [comment]
Version `0.10.0` has been released with this feature: https://pypi.python.org/pypi/sqlservice/0.10.0
